README.md

```diff
@@ -1,104 +1,52 @@
-# Deprecation notice
+# IntelliJ Rust Community Plugin
+This project is a community fork of now deprecated OpenSource [Rust plugin for Jetbrains IDEs](https://plugins.jetbrains.com/plugin/8182--come-on-dont-kill-opensource-rust) (https://github.com/intellij-rust/intellij-rust)
+This repo is not relying on Jetbrains' "c-capable" or "rust-capable" plugins, which means that it potentially might be installed onto each and any Jetbrains IDE
```

Comment:
New proprietary rust plugin is incompatible with the deprecated one (and with this one because of the same package name), and they have different names (org.rust vs com.intellij.rust IIRC) so it is not an issue. And for the deprecated one from the marketplace we can just sync their changes with this repo if there will be something useful, and then just bump version of this plugin so it would be higher than the official deprecated one.